Dockerfile

```diff
@@ -18,11 +18,11 @@
 #     docker volume create --name keywhiz-db-devel
 #
 #   Initialize the database, apply migrations, and add administrative user:
-#     docker run -v keywhiz-db-devel:/data square/keywhiz migrate
-#     docker run -it -v keywhiz-db-devel:/data square/keywhiz add-user
+#     docker run -v keywhiz-db-devel:/tmp/h2_data square/keywhiz migrate
```

Comment:
this is probably the simplest way to address the issue, but it creates a special case for making docker work with development config (if `KEYWHIZ_CONFIG` is not provided). It might be cleaner to create a new `keywhiz-development.yaml.docker` that sets H2 to use `/data` and is being used by `entry.sh` as a default.

Reply:
Yeah, I agree, that would be much cleaner way. I'll update and commit the changes.
Btw, do you usually amend changes in pull requests or commit new ones on top?

Reply:
I see. I amended and pushed for your review.

Haven't seen your changes but I'll check them. And, yeah, there should definitely be a way to verify that docker flow works properly in CI. We should consider that.